packages/api-explorer/src/Doc.jsx

```diff
@@ -289,7 +293,15 @@ class Doc extends React.Component {
         oas={this.oas}
         operation={this.getOperation()}
         formData={this.state.formData}
-        onChange={this.onChange}
+        selectedContentType={this.state.selectedContentType}
+        onChange={(e) => {
```

Comment:
What is this? Event or value? We could handle into this.onChange, and not half here and half in the method